packages/apple-pay-integration/src/mocks/apple-pay-button.mock.ts

```diff
@@ -1,8 +1,26 @@
-import { CheckoutButtonInitializeOptions } from '@bigcommerce/checkout-sdk/payment-integration-api';
+import {
+    BuyNowCartRequestBody,
+    CartSource,
+    CheckoutButtonInitializeOptions,
+} from '@bigcommerce/checkout-sdk/payment-integration-api';
 
 import { WithApplePayButtonInitializeOptions } from '../apple-pay-button-initialize-options';
 import ApplePayButtonMethodType from '../apple-pay-button-method-type';
 
+const buyNowCartRequestBody: BuyNowCartRequestBody = {
```

Comment:
You can use `getBuyNowCartRequestBody` from `@bigcommerce/checkout-sdk/payment-integrations-test-utils` instead of creating extra variable

Reply:
fixed